apps/challenges/aws_utils.py

```diff
@@ -1126,3 +1127,40 @@ def challenge_approval_callback(sender, instance, field_name, **kwargs):
                         challenge.id, failures[0]["message"]
                     )
                 )
+
+@app.task
+def export_cloudwatch_logs(submission):
+    """ 
+    This is to export cloudwatch logs of Agent and environment for a submission into S3 bucket.
+    """
+
+    for obj in serializers.deserialize("json", submission):
+        submission_obj = obj.object
+
+    try:
+        challenge = get_challenge_model(submission.challenge_phase.challenge.id)
```

Comment:
I don't think you need `try/except` while using the `get_challenge_model` function. Please take a look at its definition.